Makefile

```diff
@@ -252,14 +258,11 @@ ifeq ($(origin DEV_BANK_PRIVKEY),undefined)
 endif
 	PRIVATE_KEY=${DEV_BANK_PRIVKEY} make request-devnft recipient=${account}
 	PRIVATE_KEY=${ACCOUNT_PRIVKEY} make stake-devnft
-	PRIVATE_KEY=${ACCOUNT_PRIVKEY} make self-register-node peer_id=$(shell ./scripts/get-hopr-address.sh "$(endpoint)")
+	PRIVATE_KEY=${ACCOUNT_PRIVKEY} make self-register-node peer_id=$(shell eval ./scripts/get-hopr-address.sh "$(api_token)" "$(endpoint)")
 
 .PHONY: register-node-with-stake
 # node_api?=localhost:3001 provide endpoint of hoprd, with a default value 'localhost:3001'
-register-node-with-stake: ensure-environment-is-set
-ifeq ($(endpoint),)
-	echo "parameter <endpoint> is default to localhost:3001" >&2
-endif
+register-node-with-stake: ensure-environment-is-set, get-hopr-address-from-script
```

Comment:
The `get-hopr-address-from-script` prerequisite is non-existent.

Reply:
Thank you! Fixed!